build-i18n.bash

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+FILE_PATTERN="src/**/*.{js,jsx,ts,tsx,json}"
+
+i18next "${FILE_PATTERN}" [-oc] -c "./i18next-parser.config.js" -o "locales/\$LOCALE/\$NAMESPACE.json"
```

Comment:
Maybe we can move to use `npx`, which should come with `npm` to run this command without directly install `i18next`?

Something like: 

```bash
npx i18next "${FILE_PATTERN}" -c "./i18next-parser.config.js" -o "locales/\$LOCALE/\$NAMESPACE.json"
```

Reply:
Or add it as a script within `package.json` perhaps

Reply:
yehh thats even better :D